tests/conftest.py

```diff
@@ -20,13 +18,36 @@ def load_model_for_session(name):
 
 @pytest.fixture(scope="session")
 def simple_ode_model():
+    """ Returns the test_simple_odes.cellml model for use by testing session.
+    Note: do not use if the test attempts to modify the model.
+    """
     model = load_model_for_session('test_simple_odes')
     return model
 
 
 @pytest.fixture(scope="session")
 def bad_annotation_model():
+    """ Returns the test_bad_annotations.cellml model for use by testing session.
+    Note: do not use if the test attempts to modify the model.
+    """
     model = load_model_for_session('test_bad_annotations')
     return model
 
 
+@pytest.fixture(scope="session")
+def basic_model():
+    """ Returns the basic_ode.cellml model for use by testing session.
+    Note: do not use if the test attempts to modify the model.
+    """
+    model = load_model_for_session('basic_ode')
+    return model
+
+
+@pytest.fixture(scope="session")
+def aslanidi_model():
```

Comment:
Personally I'd stick the year in here as well. Quite a few authors have multiple

Reply:
Given the test models are sometimes modified and not 'the real thing' I think for testing this is not so vital. I can if you want but the name is really a reference to our list of cellml models in the test dir.